integration-cli/docker_cli_network_unix_test.go

```diff
@@ -236,6 +236,31 @@ func (s *DockerSuite) TestDockerNetworkDeleteNotExists(c *check.C) {
 	c.Assert(err, checker.NotNil, check.Commentf("%v", out))
 }
 
+func (s *DockerSuite) TestDockerNetworkDeleteMultiple(c *check.C) {
+	out, _, err := dockerCmdWithError("network", "rm", "test")
+	c.Assert(err, checker.NotNil, check.Commentf("%v", out))
+
+	dockerCmd(c, "network", "create", "testDelMulti")
+	assertNwIsAvailable(c, "testDelMulti")
+	dockerCmd(c, "network", "create", "testDelMulti1")
+	assertNwIsAvailable(c, "testDelMulti1")
+	dockerCmd(c, "network", "create", "testDelMulti2")
+	assertNwIsAvailable(c, "testDelMulti2")
+	dockerCmd(c, "run", "--net", "testDelMulti2", "busybox", "sh")
```

Comment:
```
out, _ := dockerCmd(c, "run", "-d", "--net=testDelMulti2", "busybox", "top")
waitRun(c, strings.TrimSpace(out))
```

This makes sure the container will stay running during the test, as it is possible to remove networks that have no running containers.